src/utils/LegacyHelpers.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  ImageStudioEvent,
+  ImageStudioEventType,
+  LegacySDKEvent,
+  SDKEvent,
+  SDKEventType,
+  SDKImageInputEventData,
+  SDKMetadata,
+} from '../types';
+
+/**
+ * translates LegacyImageStudioEvents to ImageStudioEvent
+ *
+ * @param data should be of type LegacyImageStudioEvent, but could also be any object
```

Comment:
Should this be `eventData`?
```suggestion
 * @param eventData should be of type LegacyImageStudioEvent, but could also be any object
```

Reply:
This PR had no other significant changes required, so we will save these for another time so this can be merged without re-review.